### ISSUE TITLE
HTML syntax highlighting stops showing after a certain amount of characters in one line

### ISSUE THREAD

**Issue by sthag** [adobe/brackets#11824]
I'm using Brackets Release 1.4 Build 1.4.0-16380 (release 273804eb3) on a Windows 7 and Windows 10 System.
I found this behaviour by chance. I had a select tag with a whole lot of option tags in it. I scrolled through this long line and suddenly the colors where gone.
Seems like the HTML syntax highlighting stops showing after a certain amount of characters in one line. It is only after 10000 characters so it may be no bug but intentional.
One screenshot shows the single line, the other shows the same with line break switched on.

I did not yet check if this behaviour also appears with other languages.

![moment_2015-10-15_1444912615_denzu](https://cloud.githubusercontent.com/assets/1476724/10513794/4c3ed582-734a-11e5-8a76-9330780ddd9f.png)
![moment_2015-10-15_1444912499_denzu](https://cloud.githubusercontent.com/assets/1476724/10513795/4c3fab60-734a-11e5-9096-95eb905d56ed.png)

**Comment by abose** [comment]
Brackets will stop syntax coloring on very large lines. You could break the lines into two as a workaround. The HTML semantics would not change with line breaks .

**Comment by Turqueso** [comment]
I agree with @sthag. For example, I have a long line for an inline svg wrapped in a div, brackets won't detect the last part so it wrongly thinks I did not close the div tag and highlights errors on the rest of the page when there's none because it thinks I missed a closing tag, also i can't fold this div because it won't detect the closing tag.

![image](https://cloud.githubusercontent.com/assets/4932347/11095573/f6e12414-8852-11e5-84c1-8dc60a888f16.png)
![image](https://cloud.githubusercontent.com/assets/4932347/11095618/204e5d6c-8853-11e5-9f52-aa4f3fedb929.png)
